prebid-mobile/gdpr.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Prebid Mobile Guide to European Ad Inventory and Providing Notice, Transparency and Choice
+description: European Ad Inventory Overview
+pid: 1
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile
+---
+
+
+<div class="bs-docs-section" markdown="1">
+
+# Prebid Mobile Guide to European Ad Inventory and Providing Notice, Transparency and Choice
+
+<div class="alert alert-danger" role="alert">
+  <strong>Notice</strong>
+  <p>
+  In order for publishers to meet their transparency, notice and choice/consent requirements under the GDPR and the existing ePrivacy Directive, Prebid Mobile supports <a href="http:///advertisingconsent.eu/">the IAB Europe Transparency & Consent Framework</a> (the "Framework").  
+  </p>
+  <p>
+  This is a reference for mobile app publishers using Prebid Mobile to surface notice, transparency and choice to end users located in the EEA and signal approved vendors and, where necessary, pass consent, to demand sources and their vendors.  
+  </p>
+  <p>
+  This resource should not be construed as legal advice and AppNexus  and Prebid Mobile make no guarantees about compliance with any law or regulation.  Please note that because every company and its collection, use, and storage of personal data is different, you should also seek independent legal advice relating to obligations under European regulations, including the GDPR and the existing ePrivacy Directive. Only a lawyer can provide you with legal advice specifically tailored to your situation. Nothing in this guide is intended to provide you with, or should be used as a substitute for, legal advice tailored to your business.  
+  </p>
+  <p>
+  AppNexus provides <a href="https://wiki.appnexus.com/x/DwTJAQ">Service Policies</a> (for Buying, Selling, and Data Providers) that include privacy-specific obligations of which you should be aware. These Service Policies will be updated prior to May 25, 2018.
+  </p>      
+</div>
+
+## Framework APIs
+
+Prebid Mobile provides two APIs for app publishers to use the Framework. These APIs allow you to:
+
+-   Define whether the user is located in the European Economic Area (the "EEA") and that European privacy regulations should apply
```

Comment:
What API allows you to do this?

Reply:
Which part? This was all updated by legal in the Mobile SDK version. I'll take out the part about the "approved vendor," don't know why they added that.

Reply:
Comment refers to the line "Define whether the user is located in the European Economic Area (the "EEA") and that European privacy regulations should apply".  We're are only passing location information, not resolving whether a user is in the EEA, right?

Reply:
Pre-legal review this was "Define whether GDPR should apply." So I'm not sure we're even passing location - looks like it's just true/false, use consent settings or don't. I'll change it back to what it was, substituting "European privacy regulations" for "GDPR".